Games/Stronghold/Stronghold/script.js

```diff
@@ -0,0 +1,42 @@
+include(["Functions", "QuickScript", "SteamScript"]);
+
+new SteamScript()
+    	.name("Stronghold HD")
+    	.editor("FireFly Studios")
+    	.author("odziom91")
+	.appId(40950)
+	.postInstall(function(wine, wizard) {
+		var resolutionModes = [
+	    	'800x600',
+		'1024x768',
+		'1280x720',
+		'1280x1024',
+		'1366x768',
+		'1920x1080'
+		];
+		var setupWizard = SetupWizard("Stronghold HD");
+		var selectedResolution = setupWizard.menu('Choose Virtual Desktop resolution:', resolutionModes);
+		var setupResolution = resolutionModes[selectedResolution.index];
+		switch (setupResolution) {
```

Comment:
alternatively to the switch/case block you could use:
```javascript
var resolutionParts = setupResolution.split("x");
var x = Integer.parseInt(resolutionParts[0]);
var y = Integer.parseInt(resolutionParts[1]);
wine.setVirtualDesktop(x, y);
```
This approach would be shorter and better extendable if some more resolutions are to be added.